omniauth-twitter.gemspec

```diff
@@ -16,8 +16,8 @@ Gem::Specification.new do |s|
   s.test_files    = `git ls-files -- {test,spec,features}/*`.split("\n")
   s.executables   = `git ls-files -- bin/*`.split("\n").map{ |f| File.basename(f) }
   s.require_paths = ["lib"]
-
-  s.add_dependency 'json', '~> 1.3'
+  s.required_ruby_version = Gem::Requirement.new('>= 1.9.3')
   s.add_dependency 'omniauth-oauth', '~> 1.1'
+  s.add_dependency 'rack', '~> 1.6'
```

Comment:
This version restriction for rack conflicts with that of Rails 5 (~> 2.0).  Is that intentional?